### DISCUSSION TITLE
2022-07-06 Weekly Meeting 63 Agenda - Avatar Interop Day

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#251] [Weekly Meetings]
Weekly Meeting 63 is happening on Wednesday July 6th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

- check in on two topics below (MSF delegate, and developer hour
- talk about our next official demo day occurring during our Wednesday meeting on July 27th  
- Recently we decided to take a fresh new approach to Wednesday meetings. More demos, more discussion, more exploration around weekly `theme`. Our theme for this week: `avatar-interop`.  Please feel free to chime in with any Avatar interop demos or topics you would like to discuss!

### Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

### Discussions: 

Upcoming Wednesday Calendar `themes`: :calendar: (continued in the [omigroup/media repo](https://github.com/omigroup/media/discussions/43))
- `Avatars`: Next Wednesday lets talk about Avatar Interop (July 6, 2022)
Future
- `Portals`: How should they work? What have you experimented with? What has your company been doing lately for portals? Codefrau (Croquet) said "happy to do a deep-dive into the portals I built".  (July 13, 2022)
- `Homespaces` What could interoperable homespace templating look like? Jin suggested doing workshops, and discussions on VTubing, and more.  (July 20, 2022)
- OMI DEMO DAY `Open Metaverse`  (July 27, 2022)
- `content portability`  (August 3, 2022) 

Future TBD 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo

**Comment by mrmetaverse** [comment]
Looks like you got it all in here 😊 I'll confirm and archive this prep doc: https://github.com/omigroup/omigroup/discussions/249

**Comment by mrmetaverse** [comment]
# Discussions during meetings: 
- Discord has a new feature with embedded chats in voice channels. 
- old channels like hack-chat can get confusing

**Comment by mrmetaverse** [comment]
# OMI DEMO DAY - 7/27/22
- during this meeting time (16:00UTC) 
- livecasting during meeting 
- any open metaverse topic 
- "doesn't have to be new work," you can share things you may have shared before and recently updated

**Comment by mrmetaverse** [comment]
# MSF Delegate 
- https://github.com/omigroup/omigroup/issues/250

**Comment by mrmetaverse** [comment]
# `Avatar Interop` Discussion and Demos

**Reply by mrmetaverse** [reply]
antpb shared their work on avatar creation and exporting, and a fun wordpress plugin that allows viewing the models. 
![image](https://user-images.githubusercontent.com/63426722/177599467-d14c5c7f-44e8-4884-9899-c84735acb3e4.png)
![image](https://user-images.githubusercontent.com/63426722/177599518-c0882093-3b3e-46c0-9fc6-7ca9cc7006d7.png)
![image](https://user-images.githubusercontent.com/63426722/177599620-fed9dffb-3918-4d1f-9035-3b86cc9bb74f.png)

**Reply by mrmetaverse** [reply]
considering ways to apply this as a template for omi-chan. this works in any platform that uses VRM. 
![image](https://user-images.githubusercontent.com/63426722/177599957-d1700ea6-c412-4b4d-9f5a-86f283161329.png)

**Reply by mrmetaverse** [reply]
Shirley mentioned Metahumans from Unreal: 

https://www.unrealengine.com/en-US/metahuman
![image](https://user-images.githubusercontent.com/63426722/177600260-b633fc13-252b-4901-bb4a-01eb3f4b6b52.png)

**Reply by mrmetaverse** [reply]
## Ryan with Crucible 
- shared work on a protocol for handling multiple chains 
- solutions for both VRM and FBX 
- their primary market at the moment is game developers 
- https://medium.com/@ops_25520/developing-fully-interoperable-nft-avatars-with-emergence-560e072d17b8
- 
![image](https://user-images.githubusercontent.com/63426722/177601550-3e08d5c6-ad96-40e4-8f4b-fd097aa18ec7.png)

**Reply by mrmetaverse** [reply]
- by end of this month they will have two main features available: 
  - avatar system 
  - inventory service

**Reply by mrmetaverse** [reply]
- ![image](https://user-images.githubusercontent.com/63426722/177601765-e229372d-8488-4aeb-a499-aa5ef5e87e0e.png)
- cosmetically you can change avatar and behind the scenes it will auto-change the chain based on wallet. 
- ![image](https://user-images.githubusercontent.com/63426722/177601834-b00d4301-ed78-49ce-8603-54d923aa09fc.png)

**Reply by mrmetaverse** [reply]
- touched on some of their inventory work 
- 
![image](https://user-images.githubusercontent.com/63426722/177602119-73f59918-3bc8-4740-9757-d78634e8dcf8.png)

**Reply by mrmetaverse** [reply]
- demoed ability to query and play owned music tracks through authentication (using blockchain) 
- 
![image](https://user-images.githubusercontent.com/63426722/177602618-7512537a-b0c3-4a20-87a1-28376653f2ad.png)

**Reply by mrmetaverse** [reply]
## Jin prepped a demo for todays discussion too! 
- "we are big on avatars"
- referenced [xrdevlog](https://xrdevlog.com/)
- working on ready player m3 :smile:

**Reply by mrmetaverse** [reply]
- intends for work to be super modder friendly 
- 
![image](https://user-images.githubusercontent.com/63426722/177604290-b27645b2-9856-41a6-b865-c9a6e2b59671.png)

**Reply by mrmetaverse** [reply]
- discussed Metaloot project and intent for wearables 
- 
![image](https://user-images.githubusercontent.com/63426722/177604379-2257704c-8c9b-4131-a006-1999500e9af3.png)
- 
![image](https://user-images.githubusercontent.com/63426722/177604410-15ee65f1-a51e-46d8-9ce1-5afd93d6695d.png)

**Reply by mrmetaverse** [reply]
- RTFKT unlocked the 3d avatars yesterday 
- 
![image](https://user-images.githubusercontent.com/63426722/177604487-8084837a-97fd-4c60-92db-8eb053859baa.png)
- 
![image](https://user-images.githubusercontent.com/63426722/177604525-09013fd2-1504-4479-bf22-c7a1c2e0dd73.png)
- shared some of their inventory work using both nfts and non-blockchain assets 
- 
![image](https://user-images.githubusercontent.com/63426722/177604815-cabeb7c2-6e5c-405a-bb6e-dff6c5b134d7.png)
- check out [Avatar interop group at M3](https://github.com/M3-org/avatar-interop)
- 
![image](https://user-images.githubusercontent.com/63426722/177604663-9b95f13c-fcc8-42d4-b968-00e67d6525ec.png)

**Reply by mrmetaverse** [reply]
- recently added features to dank nugz owners to incorporate in meta-data in a fun way inside of worlds like Webaverse
- made a strong case for how the web _is capable_ of more than it gets credit for.  
- discussed ability to modify avatar fidelity and swap sprites to ensure compatibility on various machines and systems 
- 
![image](https://user-images.githubusercontent.com/63426722/177605293-26d6d1d9-c846-4a64-96f4-d718afb62ad3.png)

**Reply by mrmetaverse** [reply]
# LimboFeather - WebXR just needs more fun things to do!

**Reply by mrmetaverse** [reply]
![Screenshot_20220706-130740.png](https://user-images.githubusercontent.com/63426722/177605706-9a544291-b565-40e7-9fcf-27a02b537963.png)

(Note: I had to leave but will try to capture screens of limbofeathers talk on mobile)

**Comment by mrmetaverse** [comment]
-

**Comment by mrmetaverse** [comment]
We should consider a new meeting time (test test test)
